posts/packit_1_0_0_action_required/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Packit 1.0.0 countdown - action may be required"
+date: 2024-12-09T12:00:00+00:00
+authors: mmassari
+tags:
+  - configuration
+---
+
+After 105 minor releases we are almost ready to announce our first major release, Packit 1.0.0!
+
+<!--truncate-->
+
+## What's New
+
+We will **remove all since long deprecated keys from the configuration file**.
+
+`packit-service` jobs are tight with `packit CLI` so be aware that for those projects still using deprecated keys in their `packit.yaml` project file we have created a migration pull request.
+
+Please, **check if you have any new PR for your project created by us, review and merge it**. Otherwise, you might experience issues with the `packit-service` jobs once the new packit version is released.
```

Comment:
```suggestion
Please, **check if you have any new PR for your project created by us, review and merge it**. Otherwise, you might experience issues with the `packit-service` jobs once the Packit 1.0.0 is released.
```

I would also link the user that will create the PRs in the [by us]

Reply:
I am using the packit-service tokens, see [here](https://gitlab.com/packit-service/hello-world/-/merge_requests/1298). So the user is Packit Service. Should I use different tokens? Probably I will need to create them for a different user in all gitlab.org instances...

Reply:
great, no, I think using the Packit Service user makes sense, so maybe just mention that explicitly (`created by the packit-service bot account`)